### PR TITLE
feat: Made HeroContainer component responsive

### PR DESCRIPTION
**Pull request by jainvedant392**
## Overview

1. This PR fixes or fixes part of #44.
2. This PR does the following: 
    Fixed some bugs in the styles and responsiveness of HeroContainer in Homepage

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct

image
image

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: frontend/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -25,6 +27,15 @@ const Home = () => {
           <CTA
             variant="contained"
             color="primary"
+            sx={{
```
Comment (prajjwalkapoor) [inline frontend/src/pages/Home/index.tsx:30]: Don't put sx here, instead put it in styled CTA component.
https://github.com/bsoc-bitbyte/busify/blob/d15a1b516fae2e8f80adc16ef8a30eac7f333906/frontend/src/pages/Home/index.tsx#L122

**Review by prajjwalkapoor — CHANGES_REQUESTED** [review]

**Comment by jainvedant392** [comment]
will do it by today itself

**Comment by jainvedant392** [comment]
@prajjwalkapoor, I have done the changes as requested 
please review then once

**Review by prajjwalkapoor — APPROVED** [review]
LGTM.

**Pull request merged**